starcheck/calc_ccd_temps.py

```diff
@@ -150,6 +150,19 @@ def get_ccd_temps(oflsdir, outdir='out',
     logger.info('###############################'
                 '######################################\n')
 
+    have_cxc_telem = True
+    try:
+        fetch.get_time_range('aacccdpt')
+    except KeyError:
+        logger.info("AACCCDPT not found in cheta archive.")
+        have_cxc_telem = False
+
+    use_maude = False
+    if maude == 1 or not have_cxc_telem:
```

Comment:
There is no way to set `maude` via the Python script version of this via `main()` and `get_options()`. I'd suggest adding that and writing `if maude or not have_cxc_telem:` here.

Reply:
I know you worked on the script and main() a bit for recent testing, but I have just been thinking of calc_ccd_temps.py as only useful for this function and that the script main() was historical.  Though I suppose if you are thinking about using subprocess for python from perl it starts to become more relevant again.

Reply:
I wasn't thinking about any of that. But it's faster to just make the change suggested by the reviewer and move on than to start a discussion.

Reply:
That's kind of dark.... I just didn't want to make a change that I couldn't figure out how to use or test.

Reply:
If the function is so tightly coupled to starcheck that the independent command-line interface is effectively unusable, then just cut the whole thing. But in general a command-line interface to a function should cover all the function options.

Reply:
Agreed.  I think this is a good time to cut the cmdline interface to the function.